### PR TITLE
Upload tagged images to Quay

### LINKED ISSUE #35: Release script should tag the released images
Since the release process is running E2E test with all the images, it only makes sense that we tag those images into Quay instead of every repository rebuilding it's images and tagging them.
This will ensure that the images that we release are the ones we tested and nothing unexpected slips into them (because we won't be rebuilding them).

### PR DESCRIPTION
**Pull request by mkolesnik**
The release job will upload images tagged with the current release to Quay.

Resolves #35

### CODE REVIEW

**Review by dfarrell07 — APPROVED** [review]

File: scripts/lib/image_defs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+REPO="quay.io/submariner"
+
+declare -A project_images
+project_images[lighthouse]="lighthouse-agent lighthouse-coredns"
+project_images[shipyard]="nettest shipyard-dapper-base"
+project_images[submariner]="submariner submariner-globalnet submariner-route-agent"
```
Comment (mangelajo) [inline scripts/lib/image_defs:6]:
```suggestion
project_images[submariner]="submariner submariner-globalnet submariner-route-agent submariner-networkplugin-syncer"
```
new kid on the block ':D

**Review by mangelajo — CHANGES_REQUESTED** [review]
just tiny request ':D

File: scripts/lib/image_defs (hunk 1 above)
Comment (mangelajo) [reply scripts/lib/image_defs:6]: Oh wait, that one we don't publish it yet from the submariner repo, I guess we need to update the push action

**Review by mangelajo — APPROVED** [review]

File: scripts/lib/image_defs (hunk 1 above)
Comment (mkolesnik) [reply scripts/lib/image_defs:6]: Lets update it once its being published

**Pull request merged**